version.txt

```diff
@@ -1 +1 @@
-1.11.1
+1.12.1
```

Comment:
I don't think this change deserves its own release. Let's undo this change and merge the rest.